x-pack/plugins/apm/public/components/routing/apm_route_config.tsx

```diff
@@ -63,6 +54,15 @@ const apmRoutes = route([
       }),
     ]),
   },
+  {
```

Comment:
this puts the main UI routes after the redirect routes, to ensure those are matched first (they're more specific)